wrangler.public.toml

```diff
@@ -0,0 +1,39 @@
+name = "orange-meets-public"
+
+account_id = "8477399eb04accc1792af96aeaa25222"
+main = "./build/index.js"
+# https://developers.cloudflare.com/workers/platform/compatibility-dates
+compatibility_date = "2024-06-20"
+compatibility_flags = ["nodejs_compat"]
+
+[site]
+  bucket = "./public"
+
+[[build.upload.rules]]
+  type = "ESModule"
+  globs = ["**/*.js"]
+
+[vars]
+CALLS_APP_ID = "d83fd70c193329ea4f683ffe7b34d056"
+TURN_SERVICE_ID = "46711d6c5759085dd70f599f6190ab94"
```

Comment:
Why is public the only config with a TURN_SERVICE_ID?

Reply:
Good question! I should add them to the other files — I had previously added this to other envs via the dashboard.

Reply:
Fixed!